docs/PID tuning.md

```diff
@@ -29,6 +29,24 @@ changing slowly (so the P and I terms aren't having enough impact on reaching th
 in the correction in order to reach the target sooner. If the error is rapidly converging to zero, the D term causes the
 strength of the correction to be backed off in order to avoid overshooting the target.
 
+##TPA and TPA Breakpoint
+
+* Note that TPA is set via CLI or on the PID TUNING tab of the GUI.  tpa_breakpoint is set via CLI
+
+TPA applies a 'P' value reduction in relation to full Throttle. It is used to 'soften' the P value at extream throttle which sometimes induces oscellation in the aircraft.
```

Comment:
Your help would be awesome, Petr.  I don't have a lot of technical knowledge of this stuff, and running around the net trying to gather an understanding of the basics can be challenging.  So, as I have time, I am just trying to fill in some of the gaps on the docs in CF so that the next noob can find it in one place.  I understand the difficulties involved when a lot of the coding is patched together from various fixes and such, but a basic reference is what I'm after.  Details like which PID controllers make use of this and which don't are very important so that you're not trying to tune something with values that have no effect.  So, if you and others can add that important detail, that would be great.  

I don't know if it is possible,  but it would be helpful for the user base if when a parameter is not available because of another setting, that the field in the GUI was greyed out to signify that fact.  But, that is probably a lot more coding work than just mentioning " that 'x' doesn't work if your are using 'y'. "